assets/js/index.js

```diff
@@ -7,6 +8,7 @@ require('bootstrap');
 require('select2');
 
 $(() => {
+	base();
```

Comment:
Would prefer if `base.js` would export to single methods and we would call both over here:

```
base.ieViewportFix();
base.consoleErrorFix();
```

But I’m open to other opinions.

cc/ @revrng @krnlde